app/src/debug/AndroidManifest.xml

```diff
@@ -11,7 +11,11 @@
         android:name="android.hardware.telephony"
         android:required="false" />
 
-    <application android:icon="@mipmap/ic_launcher">
+    <application
+        android:allowBackup="false"
+        android:fullBackupContent="false"
+        android:dataExtractionRules="@xml/backup_rules"
```

Comment:
I we are not allowing backup. Why do we need the backup_rules file?

Reply:
after android 12, rules are needed to include/exclude backups. In this case all are excluding so we are not allowing backups